### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by riemannulus**
- Release 0.10.0

### CODE REVIEW

**Review by earlbread — APPROVED** [review]

**Comment by dahlia** [comment]
Could we include #1050, which adjusts #1039 (naming & defaulting), as well in this release?

**Review by limebell — APPROVED** [review]

**Comment by riemannulus** [comment]
@dahlia I'll hold for that

**Review by area363 — DISMISSED** [review]

**Review by dahlia — CHANGES_REQUESTED** [review]
- [x] Could you include the patch #1050 too, which is merged today?
- [x] [As I said our Discord server][1], there seem some outdated changelogs to polish.

[1]: https://discord.com/channels/539405872346955788/596226080549896239/770170692842553384

**Comment by riemannulus** [comment]
@dahlia 👌

File: CHANGES.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Libplanet changelog
 Version 0.10.0
 --------------
 
-To be released.
+Released on October 26, 2020.
```
Comment (limebell) [inline CHANGES.md:7]:
```suggestion
Released on October 27, 2020.
```
😂

**Review by limebell — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (riemannulus) [reply CHANGES.md:7]: 😭

Comment (riemannulus) [reply CHANGES.md]: https://github.com/planetarium/libplanet/pull/1053/commits/6e2893fe5273167e97d687c37353fc2778f9e7e1 changed

**Comment by riemannulus** [comment]
@dahlia https://github.com/planetarium/libplanet/pull/1053/commits/6e2893fe5273167e97d687c37353fc2778f9e7e1 Please review again!

**Review by limebell — APPROVED** [review]

**Review by earlbread — APPROVED** [review]

**Comment by dahlia** [comment]
Instead of having a merge commit (fed5e4ef7d605c606a52a0e31efa65fcba3f536f), could you rebase this on the current main?

**Review by dahlia — APPROVED** [review]
:shipit:

**Review by moreal — APPROVED** [review]

**Review by area363 — APPROVED** [review]

**Review by earlbread — APPROVED** [review]

**Review by limebell — APPROVED** [review]

**Comment by dahlia** [comment]
The repetitive timeout on a Windows job seems the same to <https://github.com/planetarium/libplanet/pull/1050#issuecomment-716500477>.  IMHO it's okay to merge without retrying it.

**Pull request merged**